l3kernel/l3basics.dtx

```diff
@@ -1186,9 +1186,9 @@
 %     \cs{cs_if_exist:NTF} \meta{control sequence} \Arg{true code} \Arg{false code}
 %   \end{syntax}
 %   Tests whether the \meta{control sequence} is currently defined
-%   (whether as a function or another control sequence type). Any
-%   definition of \meta{control sequence} other than \tn{relax}
-%   evaluates as \texttt{true}.
+%   (whether as a function or another control sequence type),
+%   and its meaning is not the primitive \tn{relax} token
+%   (in which it differs from \cs{if_cs_exist:N}).
```

Comment:
I would change this to something like this:

... token.  This is different from \cs{if_cs_exist:N}, which evaluates to \texttt{true} if \tn{relax} if passed the token \tn{relax} as an argument.